ganghee/MyStudy2/app/src/main/java/com/example/mystudy/adapter/UpbitHolder.kt

```diff
@@ -1,25 +1,13 @@
 package com.example.mystudy.adapter
 
-import android.view.View
-import android.widget.TextView
-import androidx.core.content.ContextCompat
 import androidx.recyclerview.widget.RecyclerView
 import com.example.mystudy.data.FormatTickers
-import kotlinx.android.synthetic.main.rv_item_list.view.*
+import com.example.mystudy.databinding.RvItemListBinding
 
-class UpbitHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
-    private val coinName: TextView = itemView.tv_coin_name
-    private val tradePrice: TextView = itemView.tv_trade_price
-    private val signedChangeRate: TextView = itemView.tv_signed_change_rate
-    private val accTradePrice24th: TextView = itemView.tv_acc_trade_price_24h
+class UpbitHolder(binding: RvItemListBinding) : RecyclerView.ViewHolder(binding.root) {
+    private var rvBinding: RvItemListBinding = binding
 
     fun bind(ticker: FormatTickers) {
-        coinName.text = ticker.toMarket
-        tradePrice.text = ticker.toTradePrice
-        signedChangeRate.text = ticker.toSignedChangeRate
-        signedChangeRate.setTextColor(
-            ContextCompat.getColor(itemView.context, ticker.toRateColor)
-        )
-        accTradePrice24th.text = ticker.toAccTradePrice24h
+        rvBinding.apply { item = ticker }
```

Comment:
사소하지만.. 
```suggestion
        rvBinding.item = ticker
```
이렇게만 해도 되지않을까요?

Reply:
넵 수정하였습니다.
[UpbitHolder](https://github.com/StudyFork/GoogryAndroidArchitectureStudy/pull/195/commits/5cadb0a166cc625bf9f40cdacb7d8564d2fdf2ca)